assets/javascript/firebase.js

```diff
@@ -209,6 +229,22 @@ var firebaseBot = (function() {
     }
   }  
 
+  function updateLowGravityInfo(gameInfo) {
+    if (gameInfo.lowgravity) {
+      database.ref('games/' + gameInfo.gameId + "/playerOne/lowgravity").update({
+        lowgravity: gameInfo.lowgravity 
+      });
+    }
+  }
+
+  function updateHighGravityInfo(gameInfo) {
+    if (gameInfo.highgravity) {
+      database.ref('games/' + gameInfo.gameId + "/playerOne/highgravity").update({
```

Comment:
I believe this URL reference to the database can end at "/playerOne". It doesn't need the "/highgravity" part. It will set the highgravity property of playerOne to true, instead of another object with its own property of highgravity.